### PR TITLE
Remove unused field

### PR DESCRIPTION
**Pull request by sopel39**
no release notes needed

### CODE REVIEW

File: testing/trino-server-dev/etc/catalog/hive.properties [hunk 1]
```diff
@@ -10,8 +10,8 @@ connector.name=hive
 # Configuration appropriate for Hive as started by product test environment, e.g.
 #   trino-product-tests-launcher/bin/run-launcher env up --environment singlenode --without-trino
 # On Mac, this additionally requires that you add "<your external IP> hadoop-master" to /etc/hosts
-hive.metastore.uri=thrift://localhost:9083
-hive.hdfs.socks-proxy=localhost:1180
+hive.metastore.uri=thrift://hadoop-master:9083
```
Comment (raunaqmorarka) [inline testing/trino-server-dev/etc/catalog/hive.properties:13]: Unrelated change ?

**Review by Dith3r — APPROVED** [review]

**Review by raunaqmorarka — APPROVED** [review]
